values.yaml

```diff
@@ -100,6 +100,91 @@ expose:
 # If Harbor is deployed behind the proxy, set it as the URL of proxy
 externalURL: https://core.harbor.domain
 
+# The internal TLS used for harbor components secrue communicating. In order to enable https 
+# in each components tls cert files need to provided in advance.
+internalTLS:
+  # If internal TLS enabled
+  enabled: false
+  # There are three ways to provide tls
+  # 1) "auto" will generate cert automatically
+  # 2) "manual" need provide cert file manually in following value
+  # 3) "cert-manager" will manage cert file by cert-manager
+  certSource: "auto"
+  # The content of trust ca, only available when `certSrouce` is "manual"
+  trustCa: ""
+  # core related cert congfiguration
+  core:
+    # secret name for core's tls certs
+    secretName: "harbor-core-internal-tls"
```

Comment:
The value is also used as the name of secret if I set the `certSource` as `auto`, this will cause that I cannot deploy more than one Harbor instance in the same namespace

Reply:
secretName for `auto` and `manual` will be `{{ release-name }}-xxx`